tasmota/xsns_69_opentherm_protocol.ino

```diff
@@ -243,6 +244,14 @@ OpenThermCommand sns_opentherm_commands[] = {
      .m_ot_make_request = sns_opentherm_get_generic_u16,
      .m_ot_parse_response = sns_opentherm_parse_generic_u16,
      .m_ot_appent_telemetry = sns_opentherm_tele_generic_u16},
+    {// Boiler Lock-out Reset command
+     .m_command_name = "BLOR",
+     .m_command_code = (uint8_t)OpenThermMessageID::Command,
+     .m_flags = {.manual = 1},
```

Comment:
I like the idea of putting these one-shot commands into the list because if the send fails, the logic can retry this command and this will guarantee that BLOR will be sent when the connection is restored.

One caveat with the telemetry. Telemetry will send 0..127 if the command failed or 128..255 in case of success. Though there is no way to say if the BLOR was ever initiated or not since 0 falls under the failure scenario.

I would suggest creating a custom version of the parser. The parser might put the result code to the lower u8 and 0xFF to the upper. Also, in the case of 128..255 response range value, just replace it with 255. 

Then the stats matrix would look like this:
low:0, high: 0 - never invoked
low:0..7F, high: FF - invoked and failed, lower bit contains the boiler specific error code (I guess)
low:FF, high: FF - invoked and succeeded

Reply:
Currently `sns_opentherm_get_next_request` does not check the `manual` flag and BLOR will always be transmitted after the Tasmota re/starts. This is probably not what you want.

Instead, you can set both `notSupported` and `manual`. to True. That way `sns_opentherm_get_next_request` will always skip the BLOR.

Also, you could update `sns_opentherm_process_success_response` to have something like:

```
void sns_opentherm_process_success_response(struct OT_BOILER_STATUS_T *boilerStatus, unsigned long response)
{
    if (sns_opentherm_current_command >= SNS_OT_COMMANDS_COUNT)
    {
        return;
    }
    struct OpenThermCommandT *cmd = &sns_opentherm_commands[sns_opentherm_current_command];
    
    if (cmd->m_flags.manual) {
      // stop one-off commands from firing after confirming the request was delivered
      cmd->m_flags.notSupported = true;
    } else {
      // mark command as supported
      cmd->m_flags.supported = true;
    }
    cmd->m_ot_parse_response(cmd, boilerStatus, response);
}

```

Reply:
on every restart called sns_opentherm_protocol_reset(). It set notSupported to 1. 
I didn't see BLOR data on telemetry before manual call of BLOR